### PR TITLE
LPDs-34707 and 34709 | master

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LPD-34707
https://liferay.atlassian.net/browse/LPD-34709

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:analytics-cloud-acceptance

**Comment by fmarcio** [comment]
@interaminense @marcosapmf could you guys please review my playwright test please? Thanks

**Comment by fmarcio** [comment]
ci:test:analytics-cloud

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:analytics-cloud

**Review by interaminense — COMMENTED** [review]
Thanks, @fmarcio. I just left some questions regarding playwright tests

File: modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import {createChannel} from './utils/channel';
+import {openDataControlAndPrivacyModal} from './utils/data-control-and-privacy';
+import {ACPage, navigateTo, navigateToACPageViaURL} from './utils/navigation';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
```
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts:23]: @interaminense some time ago tests only worked if they have this featureFlagTest. There's a lot of test files with it. I can run this test without it and see if it fails or not.

File: modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test(
+	'Verify that Request Log and Suppressed Users export buttons are opening its modals when clicked',
+
+	{
+		tag: '@LPD-34707 and LPD-34709',
+	},
```
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts:33]: @interaminense it was suggested by Vini to help identify for witch task the test was required

File: modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/data-control-and-privacy.spec.ts:23]: update: I've ran my test again locally and is working without the featureFlagsTest function. I'll remove it and update this PR

**Comment by fmarcio** [comment]
resent https://github.com/liferay-ac/liferay-portal/pull/1796

**Pull request closed without merging**